authors/apps/comments/tests/test_comment_edit_history.py

```diff
@@ -0,0 +1,84 @@
+from .base_test import BaseTest
+from rest_framework import status
+from rest_framework.test import APIClient
+
+
+class TestsCommentsHistory(BaseTest):
+
+    def http_200_ok(self, response):
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def http_403_forbidden(self, response):
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def http_404_not_found(self, response):
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_retrieve_comment_history_authenticated(self):
+        """
+        Test if an authenticated user can retrieve edit history of their
+        comments
+        """
+        self.authenticate_user(self.sample_user)
+        self.create_article(self.sample_article)
+        res = self.create_comment(
+            self.sample_comment, self.sample_article)
+        comment_id = str(res.data['id'])
+        url = self.comment_url("my-data") + comment_id + '/'
+        response = self.client.put(
+            url,
+            self.sample_updated_comment,
+            format='json'
+        )
+        url = self.comment_url("my-data") + comment_id + '/' + 'history/'
+        response = self.client.get(url)
+        self.http_200_ok(response)
+        self.assertIn('Comment Edit History', response.data)
+
+    def test_retrieve_comment_history_unauthenticated(self):
+        """
+        Test if an unauthenticated user can retrieve edit history of
+        comments
+        """
+        self.authenticate_user(self.sample_user)
+        self.create_article(self.sample_article)
+        res = self.create_comment(
+            self.sample_comment, self.sample_article)
+        comment_id = str(res.data['id'])
+        url = self.comment_url("my-data") + comment_id + '/'
+        response = self.client.put(
```

Comment:
I think it's not necessary to assign this to a variable since you are not using the response